doc/user_guide/postgresql_user_guide.md

```diff
@@ -41,21 +92,109 @@ USER '<user>'
 IDENTIFIED BY '<password>';
 ```
 
+If your setup does not support SSL then simply remove suffix `?ssl=true&sslfactory=org.postgresql.ssl.DefaultJavaSSLFactory`.
+
+
+| Variable | Description |
+|----------|-------------|
+| `<host>` | Hostname or ip address of the machine hosting you PostgreSQL database. |
+| `<port>` | Port of the PostgreSQL database. Default port is `5432`, see also [Finding Out the Port of a PostgreSQL Database Installation](#finding-out-the-port-of-a-postgresql-database-installation). |
+| `<schema name>` | Name of the database schema you want to use in the PostgreSQL database. |
+
+
+## Finding Out the Port of a PostgreSQL Database Installation
+
+PostgreSQL default port is `5432`.<br />
+To inquire port in other cases use
+
+```shell
+function hfield () { head -1 | sed -e 's/  */\t/g' | cut -f $1 ; }
+SENDQ=$(ss -tl | grep postgresql | hfield 3)
+ss -tln | grep $SENDQ | hfield 4
+```
+
+## Making PostgreSQL Service Listen to External Connections
+
+See
+* https://www.bigbinary.com/blog/configure-postgresql-to-allow-remote-connection
+* https://dba.stackexchange.com/questions/83984/
+
+Use `sudo vi` to add the following line to file `/etc/postgresql/10/main/postgresql.conf`:
+```
+listen_addresses = '*'
```

Comment:
You should add a few words that you are talking about a test instance here. Otherwise you should never suggest listening on all interfaces and addresses in production.

Reply:
👍  Thanks!